kinto_client/__init__.py

```diff
@@ -50,43 +48,38 @@ def create_session(server_url=None, auth=None, session=None):
 
 
 class Endpoints(object):
+    endpoints = {
+        'root':         '{root}/',
+        'batch':        '{root}/batch',
+        'buckets':      '{root}/buckets',
+        'bucket':       '{root}/buckets/{bucket}',
+        'collections':  '{root}/buckets/{bucket}/collections',
+        'collection':   '{root}/buckets/{bucket}/collections/{collection}',
+        'records':      '{root}/buckets/{bucket}/collections/{collection}/records',      # NOQA
```

Comment:
Instead of ``# NOQA`` I would add a rule allowing longer lines for this project.

See https://github.com/nlhkabu/connect/blob/master/tox.ini#L35